.pre-commit-config.yaml

```
@@ -2,6 +2,7 @@
 #   Check out the docs at: https://pre-commit.com/
 ci:
     skip: [eslint]
+    autoupdate_schedule: quarterly
```

```suggestion
    autoupdate_schedule: monthly
```
https://pre-commit.ci/#configuration-autoupdate_schedule